msal/src/main/java/com/microsoft/identity/client/internal/controllers/MSALControllerFactory.java

```diff
@@ -88,10 +88,11 @@ public static BaseController getAcquireTokenController(@NonNull final Context ap
                                                                         @NonNull final Authority authority,
                                                                         @NonNull final PublicClientApplicationConfiguration applicationConfiguration) {
         List<BaseController> controllers = new ArrayList<>();
-        controllers.add(new LocalMSALController());
         if (brokerEligible(applicationContext, authority, applicationConfiguration)) {
-            controllers.add(new BrokerMSALController());
+            controllers.add(new BrokerMsalController());
         }
+        controllers.add(new LocalMSALController());
```

Comment:
Why does it need to add both lcoal msal controller as well when the broker msal controller is already added?

Reply:
Reverted the change back. The idea behind the design is we look in local msal first and then broker